src/org/sosy_lab/java_smt/solvers/princess/PrincessModel.java

```diff
@@ -155,6 +163,11 @@ private ImmutableList<ValueAssignment> getAssignments(
     // then handle assignments for non-array cases.
     // we expect exactly one assignment per model entry.
 
+    // FIXME: Princess may return multiply mappings for UFs if the arguments are rational, f.ex
```

Comment:
Would you be so kind and open an issue for this so that we don't forget?

Reply:
I've had another look at this and couldn't find any problems with the model. This was more of a note to myself while working on `evalImpl` where we do have to fix at least some of the values. However, this does not seem to be a problem for the model and I've now removed the note.